sql/mssql/00020-Evaluation-Tables.sql

```diff
@@ -0,0 +1,149 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+CREATE TABLE eppeta.PerformanceEvaluation (
+    EducationOrganizationId INT NOT NULL,
+    EvaluationPeriodDescriptorId INT NOT NULL,
+    PerformanceEvaluationTitle VARCHAR(50) NOT NULL,
+    PerformanceEvaluationTypeDescriptorId INT NOT NULL,
+    SchoolYear SMALLINT NOT NULL,
+    TermDescriptorId INT NOT NULL,
+    PerformanceEvaluationDescription VARCHAR(255) NULL,
+    CreateDate DATETIME2 NOT NULL,
+    LastModifiedDate DATETIME2 NOT NULL,
+    Ods_Id UNIQUEIDENTIFIER NULL,
+    Id INT IDENTITY(1,1) PRIMARY KEY,
```

Comment:
I like to be explicit about naming things.
```suggestion
    Id INT IDENTITY(1,1) NOT NULL,
    CONSTRAINT PerformanceEvaluation_PK PRIMARY KEY CLUSTERED (Id ASC)
```